libs/ledger-live-common/src/exchange/providers/index.ts

```diff
@@ -37,14 +37,14 @@ export const getProviderConfig = async (
   switch (exchangeType) {
     case ExchangeTypes.Fund:
     case ExchangeTypes.FundNg:
-      return getFundProvider(providerName.toLowerCase());
+      return getFundProvider(provider.toLowerCase());
 
     case ExchangeTypes.Sell:
     case ExchangeTypes.SellNg:
-      return await getSellProvider(providerName.toLowerCase());
+      return await getSellProvider(provider.toLowerCase());
 
     default:
-      throw new Error(`Unknown partner ${providerName} type`);
+      throw new Error(`Unknown partner ${provider} type`);
```

Comment:
Why are you still lowerCasing here and why rename to provider? is it the providerName or now the providerId ?